css/style.css

```diff
@@ -0,0 +1,402 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: system-ui, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  background-color: #f9f9f9;
```

Comment:
- [ ] apply text-align: justify so it applies to all content. If most content as justified

- [ ] add line-height property